engines/suppliers/spec/factories.rb

```diff
@@ -9,6 +9,17 @@
   end
 
   factory :order, class: Suppliers::Order do
+    confirm_before Time.now.utc
+    to_group_id do
+      group = FactoryGirl.create(:group)
+      group = Suppliers::Group.find(group.id)
+      group.id
+    end
+    from_user_id do
+      user = FactoryGirl.create(:user)
+      user = Suppliers::User.find(user.id)
```

Comment:
The more we isolate these pointless finds the easier it'll be to fix it later on